### ISSUE TITLE
Question: Which TV stick works flawlessly with this app?

### ISSUE THREAD

**Issue by Thunderbolt8** [fgl27/SmartTwitchTV#205]
Not an issue, bug or request, but merely a question. I dont know where else I would ask this where are potentially so many people having experience with this app:

Which TV stick would you recommend in order to be able to use the Android app flawlessly (no lags, dropped frames with chat open etc) at 1080p60 source formats without any hiccups etc.? (My (other) TV is too old/incompatible for the app to run on it at all)

Does the Amazon latest FireTV stick do the job? I dont mind having to install few additional programs on top as long as the app would work in the end (and I could still use the stick normally for its intended job as well). 

Or would you rather recommend a pure Android TV stick for this? If so, which one? or which kind of technical specs (CPU/GPU?) should the stick have?

I'd hate having to buy a TV stick randomly only to find out its not compatible with the app. Thanks for helping out.

(sorry if this is misplaced here)

**Comment by fgl27** [comment]
I don't have any stick, I do own a Nvidia shield.

Amazon are usually buged if not they will be soon as they support sucks, they performance is on low low end.

Chinese sticks are on the same as Amazon.

Google one are the second best as they work and have proper support, but usually are on the low end of performance.

Nvidia are the best but more expensive, they have the best support and performance, but they aren't sticks the are a little big.

**Comment by fgl27** [comment]
Most TVs are on the same as Google, but the support varies from vendor to vendor, can't advise any as I don't have and see bugs here and there from it.

Even the most expensive Sony do have issues.

**Comment by Thunderbolt8** [comment]
so nvidia shield boxes are based on android as well (or at least they are compatible)? they are a bit on the pricey side, though (and no AV1 support it seems).

what about such generic android boxes? (sorry, german amazon)

https://www.amazon.de/Android-Allwinner-Support-Dual-WiFi-Fernbedienung-Schwarz/dp/B08QJ2FZYX
https://www.amazon.de/Allwinner-Quad-Core-64-Bit-ARM-Corter-A53-CPU-Mini-Tastatur-Schwarz/dp/B0B8ZQB77R

**Comment by fgl27** [comment]
Yes I use the 2017 shield as main device it runs Android 11
And it can run anything, 5 streams at the same time so 1080p60

Only Android device capable of that 

Is price I bought my second hand by half price.

It will never get av1 but also twitch may never do, even if twitch does they will keep supporting avc.

Those Chinese boxes work but when they don't you have zero support.

At least they are cheap so you may not care of they malfunction.

Never used those only even cheaper ones, they work but there is some lag here and there.

**Comment by Thunderbolt8** [comment]
Do you know if you can further tweak the shield so that other android apps can run in it as well? Like the Google play store (and apps from there) and SmartTube?

**Comment by fgl27** [comment]
Yes it's a official Android device.

Has tv play store and you can very easily install any APK even phone apps, as the TV store is limited I often do.

As long you have a mouse you can use phone apps that only support touch.

I use a remote with air mouse support.

Any TV made app works flowless with normal remote

**Comment by Thunderbolt8** [comment]
Does the shield have running noise (Fans etc.)?

**Comment by Thunderbolt8** [comment]
do you mean the standing laptop like model with "box"? its not a real cube though, right?

**Comment by fgl27** [comment]
Yes the older model look, is a regular shape more like a set top box, this one has an fan, with is completely silent
